### ISSUE TITLE
Navigation issue with the Staking page

### ISSUE THREAD

**Issue by ipopescu** [casper-network/docs#100]
Go to the Staking page: https://casper.network/docs/staking.
Click on one of the links: "How to Stake your CSPR" or "Delegating with the Command-line".
You will see that when you click, some other links are getting pulled from the https://casper.network/docs/workflow/staking section. 
Figure out how to organize these pages. The left navigation pane for the Staking section should not bring in the Workflow/staking links.

https://user-images.githubusercontent.com/4185994/143627262-030cd32f-d597-4fba-abac-a6c54de3d86d.mov

**Comment by ipopescu** [comment]
Issue assigned to sd-ditoy. Removing the high-priority flag since it will be worked on.

**Comment by ipopescu** [comment]
From Sandro: This issue seems to stem from the fact that the pages:

/docs/workflow/staking
/docs/workflow/delegate
are part of the /workflow section / folder but are referenced in the /staking section.
It appears that docusaurus will assume a page is "rooted" in what folder it resides, so it belongs to /workflow. Thereby once you navigate to that doc/article it will show the navigation for that section/folder

The easiest way to fix this, but also the most undesirable one, would be to have unique pages for both sections of course. Which would mean duplication.

I have tried a couple attempts at "importing" the content for example from /docs/workflow/staking into a page called /staking/howto. By creating a empty page that just references the original (see: https://docusaurus.io/docs/next/markdown-features/react#importing-markdown )

This however has side-effects. For example the Navigation on the righthand-side is not rendered AND links within the document to other relative based documents are broken.

I have looked if there are existing solutions/plugins for this kind of use-case. However the ONE and only existing one that i have found ( https://github.com/dotansimha/remark-import-partial ) exhibits the same or similar issues.

So based on what we know now. I see two options on how to proceed:

Duplicate the pages and manually edit the links to point to the correct targets.
Develop a custom plugin that could cover this use case.
The Custom Plugin approach would mean to develop a plugin that covers atleast the following features:

Read content from a referenced .md file and import it to this new file
Ensure Navigation can be rendered correctly
Rewrite/Transform any existing relative links to their appropriate destination
I say atleast, because i might not see some hidden requirements that may pop up. As i'm fairly new to this project and its dependencies/quirks. This in turn obviously also means that is very hard to estimate the time i would have to consume to develop this feature/plugin.

Let me know what you think and how we should proceed. Oh, and of course if something is unclear, we can gladly schedule a call/meeting to discuss these issues further.
